google-cloud-clients/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/models/RowMutation.java

```diff
@@ -37,18 +37,50 @@
   private final ByteString key;
   private final Mutation mutation;
 
-  private RowMutation(String tableId, ByteString key) {
+  private RowMutation(String tableId, ByteString key, Mutation mutation) {
     this.tableId = tableId;
     this.key = key;
-    this.mutation = Mutation.create();
+    this.mutation = mutation;
   }
 
+  /** Creates a new instance of the mutation builder. */
   public static RowMutation create(@Nonnull String tableId, @Nonnull String key) {
     return create(tableId, ByteString.copyFromUtf8(key));
   }
 
+  /** Creates a new instance of the mutation builder. */
   public static RowMutation create(@Nonnull String tableId, @Nonnull ByteString key) {
-    return new RowMutation(tableId, key);
+    return new RowMutation(tableId, key, Mutation.create());
+  }
+
+  /**
+   * Creates new instance of mutation builder by wrapping existing mutation builder.
+   *
+   * <p>Sample code:
+   *
+   * <pre><code>
+   * Mutation mutation = Mutation.create()
+   *     .setCell("[FAMILY_NAME]", "[QUALIFIER]", [TIMESTAMP], "[VALUE]");
+   * RowMutation rowMutation = RowMutation.create("[TABLE]", "[ROW_KEY]", mutation);
+   * </code></pre>
+   */
+  public static RowMutation create(@Nonnull String tableId, @Nonnull String key, @Nonnull Mutation mutation) {
+    return create(tableId, ByteString.copyFromUtf8(key), mutation);
+  }
+
+  /**
+   * Creates new instance of mutation builder by wrapping existing mutation builder.
```

Comment:
Creates new instance of mutation builder by wrapping existing existing set of row mutations. The builder will be owned by this RowMutation and should not be used by the caller after this call. This functionality is intended for advanced usage.